docs/core/v3.0.rst

```diff
@@ -488,101 +486,48 @@ Core data types
 
```

Comment:
IMO a note about the little-endian defaults which can be changed via the `Endian` codec would be great in the `Data types` section above.

Reply:
Done.